linux/Makefile-manual

```diff
@@ -44,6 +44,6 @@ $(CPPOBJS): %.o: %.cpp
 
 
 clean:
-	rm -f $(OBJS) hidtest-hidraw libhidapi-hidraw.so ../hidtest/hidtest.o
+	rm -f $(OBJS) hidtest-hidraw libhidapi-hidraw.so $(CPPOBJS)
```

Comment:
CPPOBJS is empty, I think COBJS should be here

Reply:
Exact.
New version pushed.